troposphere/elasticsearch.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2013, David Irvine <irvined@gmail.com>
+# All rights reserved.
+#
+# See LICENSE file for full license.
+
+from . import AWSObject, Ref, GetAtt, Tags, AWSProperty
+from .validators import boolean, integer
+
+
+class ElasticSearchEBSOption(AWSProperty):
```

Comment:
I would drop the `ElasticSearch` in all of these class names.  The module path should handle that for us.  So this would become just `EBSOptions`